libflux/flux-core/src/formatter/mod.rs

```diff
@@ -329,24 +346,84 @@ impl<'doc> Formatter<'doc> {
+            Node::Identifier(x) => self.format_identifier(x),
+            Node::ArrayExpr(x) => self.format_array_expression(x),
+            Node::DictExpr(x) => self.format_dict_expression(x),
+            Node::FunctionExpr(x) => self.format_function_expression(x),
+            Node::LogicalExpr(x) => self.format_logical_expression(x),
+            Node::ObjectExpr(x) => self.format_object_expression(x),
+            Node::MemberExpr(x) => self.format_member_expression(x),
+            Node::IndexExpr(x) => self.format_index_expression(x),
+            Node::BinaryExpr(x) => self.format_binary_expression(x),
+            Node::UnaryExpr(x) => self.format_unary_expression(x),
+            Node::PipeExpr(x) => self.format_pipe_expression(x),
+            Node::CallExpr(x) => self.format_call_expression(x),
+            Node::ConditionalExpr(x) => self.format_conditional_expression(x),
+            Node::StringExpr(x) => self.format_string_expression(x),
+            Node::ParenExpr(x) => self.format_paren_expression(x),
+            Node::IntegerLit(x) => self.format_integer_literal(x),
+            Node::FloatLit(x) => self.format_float_literal(x),
+            Node::StringLit(x) => self.format_string_literal(x),
+            Node::DurationLit(x) => self.format_duration_literal(x),
+            Node::UintLit(x) => self.format_uint_literal(x),
+            Node::BooleanLit(x) => self.format_boolean_literal(x),
+            Node::DateTimeLit(x) => self.format_date_time_literal(x),
+            Node::RegexpLit(x) => self.format_regexp_literal(x),
+            Node::PipeLit(x) => self.format_pipe_literal(x),
+            Node::ExprStmt(x) => self.format_expression_statement(x),
+            Node::OptionStmt(x) => self.format_option_statement(x),
+            Node::ReturnStmt(x) => self.format_return_statement(x),
+            Node::TestCaseStmt(x) => self.format_testcase(x),
+            Node::BuiltinStmt(x) => self.format_builtin(x),
+            Node::Block(x) => self.format_block(x),
+            Node::Property(x) => self.format_property(x),
+            Node::TextPart(x) => self.format_text_part(x),
+            Node::InterpolatedPart(x) => self.format_interpolated_part(x),
+            Node::VariableAssgn(x) => self.format_variable_assignment(x),
+            Node::MemberAssgn(x) => self.format_member_assignment(x),
+            Node::TypeExpression(x) => self.format_type_expression(x),
+            Node::MonoType(x) => self.format_monotype(x),
+            Node::PropertyType(x) => self.format_property_type(x),
+            Node::ParameterType(x) => self.format_parameter_type(x),
+            Node::TypeConstraint(x) => self.format_type_constraint(x),
+            Node::BadExpr(_) | Node::BadStmt(_) | Node::Package(_) => {
+                return Err(anyhow!("Unable to format node"))
+            }
+        })
+    }
+
+    fn format_package_clause(&mut self, pkg: &'doc ast::PackageClause) -> Doc<'doc> {
```

Comment:
All changes from here on in this file are just boring extraction of code into functions so I could call them in `format_node`